tsconfig.build.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
why do we need typescript project references here? wouldn't a single tsconfig suffice?

Reply:
gotcha